lexibank_kpaamcam.py

```diff
@@ -1,38 +1,44 @@
-from pathlib import Path
+import pathlib
+import collections
+
 from clldutils.misc import slug
-from pylexibank import FormSpec, Lexeme, Concept, Language
+from pylexibank import FormSpec
 from pylexibank import Dataset as BaseDataset
-from pylexibank import progressbar
-from lingpy import *
-import attr
-
-@attr.s
-class CustomConcept(Concept):
-    PartOfSpeech = attr.ib(default=None)
```

Comment:
Custom object classes are only necessary when the dataset has non-standard fields. So we could use a custom `Language` to add the speaker metadata - if we want to model each speaker as doculect.